doc/Doxyfile.in

```diff
@@ -564,7 +564,7 @@ WARN_LOGFILE           = @PROJECT_BINARY_DIR@/DoxyWarnings.log
 # directories like "/usr/src/myproject". Separate the files or directories 
 # with spaces.
 
-INPUT                  = @IFEM_PATH@/src \
+INPUT                  = @IFEM_PATH@/src @IFEM_PATH@/Release \
```

Comment:
dette blir vel feil, det skal være @CMAKE_BINARY_DIR@ og den skal da være der fra før ?

Reply:
Jeg måtte ha denne for at den skulle finne IFEM.h (som blir generert fra IFEM.h.in) når jeg kjører make doc under f.eks. Elasticity/Linear/Release og ikke bruker IFEM_AS_SUBMODULE. CMAKE_BINARY_DIR vil vel da peke på den app-spesifikke Release-folderen, som ikke inneholder IFEM.h, sermed får man doxy-warningsene. Mulig det finnes en mer elegant løsning på dette?

Reply:
ah, da vil nok @PROJECT_BINARY_DIR@ være mer riktig.

Reply:
ah ok. jeg setter en dedikert variabel. bruk @IFEM_H_PATH@

Reply:
Endret til `@IFEM_H_PATH@`. Det virker.